src/sql/parts/notebook/notebookActions.ts

```diff
@@ -53,6 +54,19 @@ export class AddCellAction extends Action {
 	}
 }
 
+
+// Action to clear outputs of all code cells.
```

Comment:
outputs --> results

Reply:
Same as above comment as we are using 'output' in codebase